torch/nn/quantized/modules/linear.py

```diff
@@ -42,3 +50,71 @@ def __init__(self, in_features, out_features, bias=True):
     def forward(self, x):
         Y_q = torch.ops.quantized.fbgemm_linear(x, self._packed_weight, self.bias, self.output_scale, self.output_zero_point)
         return Y_q
+
+    def state_dict(self, destination=None, prefix='', keep_vars=False):
```

Comment:
https://github.com/pytorch/pytorch/pull/21933 might be helpful

Reply:
@dzhulgakov sure, I'll change after that is landed.